apps/connect-v1/src/providers/telemetry.ts

```diff
@@ -75,6 +80,8 @@ export const eventHandler = (e: WormholeConnectEvent) => {
     txId: e.details.txId,
     USDAmount: e.details.USDAmount,
     amount: e.details.amount,
+    connectVersion: e.meta?.version,
+    connectHost: e.meta ? (window.location.host + window.location.pathname) : undefined,
```

Comment:
maybe this could be just 

```suggestion
    connectHost:  `${window.location.host}${window.location.pathname}`,
```

or does it matters if e.meta is defined or not?

Reply:
I put that condition for some validation in the test, but I think I can remove the connectHost file, Miguel told me with the version is enough https://xlabs-org.slack.com/archives/C068BM70P3Q/p1728593255758959?thread_ts=1728497213.197049&cid=C068BM70P3Q